src/main/java/org/eclipse/xpanse/terraform/boot/api/TerraformApiController.java

```diff
@@ -184,42 +182,6 @@ public TerraformResult destroyWithScripts(
         return terraformScriptsService.destroyWithScripts(request, uuid);
     }
 
-    /**
```

Comment:
@WangLiNaruto there is no need to remove these methods. These methods are not used from xpanse but terraform-boot is a generic project and this method can be used by other projects in future.